client/js/table-view.js

```diff
@@ -62,13 +63,49 @@ class TableView {
       }
       docFragment.appendChild(tr);
     }
-    removeChildren(this.bodyEle);
-    this.bodyEle.appendChild(docFragment);
+    removeChildren(this.bodyEl);
+    this.bodyEl.appendChild(docFragment);
+  }
+
+  renderTableFoot() {
+    removeChildren(this.footEl);
+    const docFragment = document.createDocumentFragment();
+    const tr = createTR();
+    for(let col = 0; col < this.model.numOfCols; col++) {
+      let sum = this.calculateColSum(col);
+      if(sum === 0) {
+        sum = '';
+      }
+      const td = createTD(sum.toString());
+      tr.appendChild(td);
+    }
+    docFragment.appendChild(tr);
+    this.footEl.appendChild(docFragment);
+    this.setFootColor();
+  }
+
+  calculateColSum(col) {
+    const trs = document.querySelectorAll('TBODY TR');
+    return Array.from(trs)
+      .map(function(tr) { 
```

Comment:
Do you suggest moving 'calculateColSum' into one of the util files(dom-util/array-util) ?